### DISCUSSION TITLE
I can find two transfer of XCH to my wallet address from chia explorer  but shows only one in my local wallet, even after many days and Synced

### DISCUSSION THREAD

**Discussion by kugewang2022** [Chia-Network/chia-blockchain#6064] [Support]
as photo shows
![QQ图片20210526222720](https://user-images.githubusercontent.com/84851484/119659773-dc356200-be71-11eb-8d66-429165640701.png)
![QQ图片20210526222958](https://user-images.githubusercontent.com/84851484/119659836-ece5d800-be71-11eb-9284-35d68e699d8e.png)

**Comment by tiny0little** [comment]
your node and wallet in sync?

**Comment by kugewang2022** [comment]
![QQ图片20210526225325](https://user-images.githubusercontent.com/84851484/119662861-31bf3e00-be75-11eb-9e48-5705b6676d13.png)
as it shows, I believe its already synced

**Reply by skypro1111** [reply]
you need to wait for the synchronization of the wallet
![image](https://user-images.githubusercontent.com/6317389/119663861-7201ca80-be3b-11eb-8602-74c3549b10e2.png)

**Reply by kugewang2022** [reply]
but as we can see there, synced means it's been done the synchronization right?

**Reply by skypro1111** [reply]
I did not fully understand the mechanism, but in reality you need to wait for the wallet to sync to the block in which your transaction was carried out

**Comment by IsraBNC** [comment]
Same problem here. I can see the transfer done on chia explorer but I never received it on the chia wallet. What im missing?

**Reply by kugewang2022** [reply]
well, it's good to know im not alone lol

**Reply by IsraBNC** [reply]
XD

**Comment by DoniniII** [comment]
Same problem here. I can see the transfer done on chia explorer but I never received it on the chia wallet.

**Comment by thewouche** [comment]
same problem here. It took me a 9 days to sync the full node, however the wallet still not synced and i can't see my XCH that i bought and transferred in my wallet. I can see the transfer done on explorer but I never received it on the chia wallet...PLEASE HELP..

**Comment by XIII39** [comment]
I had the same problem this morning, the transaction was on block 285,XXX (I don't remember exactly) and my wallet was synced well beyond that. I did a View -- > Force Reload and when my wallet loaded back up, everything was where it was supposed to be.

**Comment by BrandtH22** [comment]
@kugewang2022 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
